app/views/application/_navbar.html.erb

```diff
@@ -53,7 +53,7 @@
         <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12">
           <ul class="nav nav-pills actions-menu">
             <%= render 'application/menus/user_list_link' %>
-            <%= render 'application/menus/offers_dashboard_link' %>
+            <%= render 'application/menus/offers_link' %>
```

Comment:
Maybe `offers_list_link` for consistency with the other partials: `user_list_link`, `inquiries_list_link`